src/styles/utils/variable.scss

```diff
@@ -0,0 +1,17 @@
+$width-days: 100px;
+$hight-days: 100px;
```

Comment:
There seems to be a typo in the variable name `$hight-days`. According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), variable names should be spelled correctly. It should probably be `$height-days`.